bubble_sort.rb

```diff
@@ -0,0 +1,33 @@
+def bubble_sort(arr)
+	def swap(val1, val2, arr, index)
+		arr[index+1] = val1
+		arr[index] = val2
+	end
+	n = arr.length-1
+	k = n
+	n.times do
+		arr.each_with_index do |value, index|
```

Comment:
If you only want to use the index you can try with [`each_index`](https://ruby-doc.org/core-2.6/Array.html#method-i-each_index)